ntc_templates/templates/alcatel_aos_show_interfaces_ethernet.textfsm

```diff
@@ -0,0 +1,86 @@
+Value PORT (\S+)
+Value STATUS (\S+)
+Value ADMIN_STATUS (\S+)
+Value PORT_DOWN_VIOLATION_REASON (\S.+\S)
+Value LAST_TIME_LINK_CHANGED (\S.+\S)
+Value NUMBER_OF_STATUS_CHANGE (\d+)
+Value PORT_TYPE (\S.+\S)
+Value SFP_XFP (\S.+\S)
+Value INTERFACE_TYPE (\S.+\S)
+Value ETHERNET_POWER_POLICY (\S.+\S)
+Value LINK_QUALITY (\S.+\S)
+Value MAC_ADDRESS (\S.+\S)
+Value BANDWIDTH (\d+)
+Value DUPLEX (\S+)
+Value AUTONEGOTIATION (\S.+\S)
+Value LONG_FRAME_SIZE (\d+)
+Value INTER_FRAME_GAP (\d+)
+Value LOOPBACK_MODE (\S.+\S)
+Value RX_BYTES_RECEIVED (\d+)
+Value RX_BROADCAST_FRAMES (\d+)
+Value RX_UNDERSIZE_FRAMES (\d+)
+Value RX_LOST_FRAMES (\d+)
+Value RX_CRC_ERROR_FRAMES (\d+)
+Value RX_UNICAST_FRAMES (\d+)
+Value RX_MULTICAST_FRAMES (\d+)
+Value RX_OVERSIZE_FRAMES (\d+)
+Value RX_ERROR_FRAMES (\d+)
+Value RX_ALIGNMENT_ERRORS (\d+)
+Value TX_BYTES_TRANSMITTED (\d+)
+Value TX_BROADCAST_FRAMES (\d+)
+Value TX_UNDERSIZE_FRAMES (\d+)
+Value TX_LOST_FRAMES (\d+)
+Value TX_ERROR_FRAMES (\d+)
+Value TX_UNICAST_FRAMES (\d+)
+Value TX_MULTICAST_FRAMES (\d+)
+Value TX_OVERSIZE_FRAMES (\d+)
+Value TX_COLLIDED_FRAMES (\d+)
+Value TX_COLLISIONS (\d+)
+Value TX_LATE_COLLISIONS (\d+)
+Value TX_EXCESSIVE_COLLISIONS (\d+)
+
+
+Start
+# R6
+  ^\s*Slot\/Port\s*${PORT}\s*:\s*$$
+# R8
+  ^\s*Chassis\/Slot\/Port\s*:\s*${PORT}\s*$$
```

Comment:
@evilmonkey19
Does it make sense to consolidate the R6 and R8 regexes?
(If you don't think so, that's ok.)

```suggestion
^\s*(Chassis\/)?Slot\/Port\s*(:\s*)?${PORT}\s*(:\s*)?$$
```